draft-js-plugins-editor/src/Editor/createCompositeDecorator.js

```diff
@@ -1,18 +1,23 @@
+// @flow
+
 /**
  * Creates a composite decorator based on the provided plugins
  */
 
 import { List } from 'immutable';
-import { CompositeDecorator } from 'draft-js';
+import { CompositeDecorator, EditorState } from 'draft-js';
+import { type DraftDecorator } from 'draft-js/lib/DraftDecorator';
 import decorateComponentWithProps from 'decorate-component-with-props';
 
-export default (decorators, getEditorState, setEditorState) => {
-  const convertedDecorators = List(decorators)
+export default (
+  decorators: List<DraftDecorator>,
+  getEditorState: () => EditorState,
+  setEditorState: (editorState: EditorState) => void) => {
+  const convertedDecorators = decorators
     .map((decorator) => ({
       ...decorator,
       component: decorateComponentWithProps(decorator.component, { getEditorState, setEditorState }),
-    }))
-    .toJS();
+    })).toArray();
```

Comment:
I'm not 100% sure, but afaik toJS does a deep conversion and toArray only a shallow one. Is this desired here?